src/main/java/sw_10/p3_backend/Logic/BladeTaskLogic.java

```diff
@@ -27,8 +28,10 @@ public class BladeTaskLogic {
     private final BookingLogic bookingLogic;
     private final ResourceOrderLogic resourceOrderLogic;
     private final BladeProjectLogic bladeProjectLogic;
-    private final ConflictLogic conflictLogic;
+    private final BookingRepository bookingRepository;
 
+    @Autowired
+    private ConflictLogic conflictLogic;
```

Comment:
hvorfor autowire du den når den bliver fixed med en dependency injection

Reply:
Det er for at fikse cirkulære dependencies, siden Spring Boot ikke kan finde en rækkefælge at injecte dependencies, når de skal bruge hinanden